### PR TITLE
Image and text storing and loading

### PR DESCRIPTION
**Pull request by cryptoquick**
Can now store text and images
Can now view text and images
Renamed Content page to View page
Added some fast-build config

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Rename this file to `config.toml` to enable "fast build" configuration. Please read the notes below.
```
Comment (kn0wmad) [inline .cargo/config.toml:1]: I'm guessing nightly is the more useful build during development, and we can move to stable rust for production builds?

File: README.md [hunk 2]
```diff
@@ -47,3 +45,25 @@ Feel free to send some Monero to the address below. And if you do so, feel free
 XMR: `8ADbBKaunVWjdg5aWQ5ZBNDACdPVMTUBnKETaZbUZ8gMfDfpwhcBeo31kfUgCJKATMPaqmsUoxBwicTpRLg4p4F57kPJ5ab`
 
 For more, see our [Contributions document](CONTRIBUTING.md)
+
+## Building
```
Comment (kn0wmad) [inline README.md:49]: Build instructions - Niiiice!

**Review by kn0wmad — COMMENTED** [review]
Looks good, can't wait til I know half as much rust as you

File: .cargo/config.toml (hunk 1 above)
Comment (cryptoquick) [reply .cargo/config.toml:1]: Generally, that is the plan, yes. Much faster builds on nightly. Nightly builds are sometimes faster and smaller, too. It's not generally a bad idea to use it imo, honestly, but also, things could break in unexpected ways. If we ever suspect that's the case, building with `cargo +stable run` could help to investigate that.

File: README.md (hunk 2 above)
Comment (cryptoquick) [reply README.md:49]: Honestly, I want to include a bit of a "beginner's guide" at some point, linking to good resources to learn more about various things Fuzzr does and uses, links to documentation for a very beginner to learn how to check out git repos, and basic platform-specific instructions on how to build Fuzzr for their respective platforms. Some basic troubleshooting steps could be written, also. I'll make an issue for this if I remember.

Comment (kn0wmad) [reply README.md:49]: And of course, eventually system-specific binaries will make thinks even easier

**Pull request merged**